frontend/packages/dev-console/src/components/import/image-search/ImageStreamTagDropdown.tsx

```diff
@@ -111,6 +111,7 @@ const ImageStreamTagDropdown: React.FC = () => {
       label="Tag"
       items={imageStreamTagList}
       key={imageStream.image}
+      autoComplete
```

Comment:
Import `fuzzy` then:
```suggestion
      autocompleteFilter={fuzzy}
```

Reply:
updated